pages/_document.tsx

```diff
@@ -9,6 +9,7 @@ class DualPowerAppDocument extends Document {
             href='https://fonts.googleapis.com/css2?family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap'
             rel='stylesheet'
           />
+          <link href='https://rsms.me/inter/inter.css' rel='stylesheet' />
```

Comment:
Are we adding 'inter' as the font we want to use generally? I fear I may have missed this memo :wink:

Reply:
I saw the designs were using inter font, so I just threw it in.

Reply:
@jackhajb those are wireframes, not mockups. the brand sans is helvetica now

Reply:
@replygirl Seemed pretty detailed and assumed we were working from them as mockups.  Noted.